src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -64,17 +89,17 @@ object Lorre extends App with LazyLogging {
   /**
     * Fetches and stores all accounts from the latest block stored in the database.
     */
-  def processTezosAccounts(): Try[Unit] = {
+  def processTezosAccounts(network: String): Try[Unit] = {
     logger.info("Processing latest Tezos accounts data..")
-    tezosNodeOperator.getLatestAccounts("zeronet") match {
+    tezosNodeOperator.getLatestAccounts(network) match {
       case Success(accountsInfo) =>
         Try {
           val dbFut = TezosDatabaseOperations.writeAccountsToDatabase(accountsInfo, db)
           dbFut onComplete {
             case Success(_) => logger.info(s"Wrote ${accountsInfo.accounts.size} accounts to the database.")
             case Failure(e) => logger.error(s"Could not write accounts to the database because $e")
           }
-          Await.result(dbFut, Duration.Inf)
+          Await.result(dbFut, Duration.apply(awaitTimeInSeconds, SECONDS))
```

Comment:
this can be simplified everywhere you happen to notice using

```scala
import scala.concurrent.duration._ //import
....
//profit
Await.result(dbFut, awaitTimeInSeconds.seconds)
```